packages/language-service/src/typescript_symbols.ts

```diff
@@ -608,15 +608,10 @@ class PipeSymbol implements Symbol {
         let resultType: ts.Type|undefined = undefined;
         switch (this.name) {
           case 'async':
-            switch (parameterType.name) {
-              case 'Observable':
-              case 'Promise':
-              case 'EventEmitter':
-                resultType = getTypeParameterOf(parameterType.tsType, parameterType.name);
-                break;
-              default:
-                resultType = getTsTypeFromBuiltinType(BuiltinType.Any, this.context);
-                break;
+            // It's safe to use Symbol here for the type 'Observable, Promise, EventEmitter'.
```

Comment:
Could you rephrase or elaborate this comment? What do you mean by safe?
```suggestion
            // It's safe to use Symbol here for the type 'Observable', 'Promise', or 'EventEmitter'.
```

Reply:
this place only needs to handle these three types. It's enough to use Symbol. Maybe replace it with 'enough'.

Reply:
Consider rephrasing to `Get symbol of 'Observable', 'Promise', or 'EventEmitter' type` or similar.

Reply:
How about `Get TypeParameter of 'Observable', 'Promise', or 'EventEmitter' type`